doc/source/whatsnew/v0.25.0.rst

```diff
@@ -28,7 +28,38 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- :meth:`Timestamp.strptime` will now rise a NotImplementedError (:issue:`21257`)
```

Comment:
Fixed a typo here ("rise" -> "raise"), which is why the diff shifts it down

Reply:
Actually, the typo was not the only error here. The issue should be #25016, PR was #25124.

Might be worth fixing separately, but I've also fixed it here.

@mroeschke @saurav2608

Reply:
Actually, the typo was not the only error here. The issue should be #25016, PR was #25124.

Might be worth fixing separately, but I've also fixed it here.

@mroeschke @saurav2608

Reply:
@h-vetinari - thanks. I will correct it independent of this.